ross/tests/test_rotor_assembly.py

```diff
@@ -851,91 +851,72 @@ def test_static_analysis_rotor5(rotor5):
 
 @pytest.fixture
 def rotor6():
-    #  Overhung rotor without damping with 10 shaft elements
```

Comment:
Maybe we should keep this test and add an additional test for the coaxial rotor.

Reply:
Done.